lib/telemetry_metrics_statsd/event_handler.ex

```diff
@@ -40,19 +42,21 @@ defmodule TelemetryMetricsStatsd.EventHandler do
   end
 
   def handle_event(_event, measurements, metadata, %{
-         reporter: reporter,
-         metrics: metrics,
-         mtu: mtu,
-         prefix: prefix
-       }) do
+        reporter: reporter,
+        metrics: metrics,
+        mtu: mtu,
+        prefix: prefix,
+        formatter: formatter_mod
+      }) do
     packets =
       for metric <- metrics do
         case fetch_measurement(metric, measurements) do
           {:ok, value} ->
             # The order of tags needs to be preserved so that the final metric name is built correctly.
             tag_values = metric.tag_values.(metadata)
             tags = Enum.map(metric.tags, &{&1, Map.fetch!(tag_values, &1)})
-            Formatter.format(prefix, metric, value, tags)
+            normalized_name = add_prefix_to_metric_name(prefix, metric.name)
```

Comment:
@arkgil I took a stab at the metric name normalization. Not too sold on the name, but this does work to make the formatter not need to worry about it.